R/utils-html.R

```diff
@@ -193,7 +193,7 @@ getComments <- function(comments) {
 #' @param inQualifiers data frame of filtered (for SVP) or all (for SRS) qualifiers.
 #' @return list of deduplicated qualifiers with column names.
 formatQualifiersTable <- function(inQualifiers) {
-    if (isEmptyOrBlank(inQualifiers) || nrow(inQualifiers) == 0) return()
+    if(isEmptyOrBlank(inQualifiers) || (isEmptyVar(inQualifiers))) return()
```

Comment:
One thing that Lindsay suggested on my stuff was to limit functions to only having a single return. I guess maybe that should be the case here? Maybe wrap the rest of the function in a negated version of this if statement?